editor/src/components/inspector/sections/layout-section/layout-system-subsection/split-chained-number-input.tsx

```diff
@@ -214,6 +222,95 @@ const handleSplitChainedEvent =
       unsetProperty(element, longhand.L),
     ]
 
+    function actionsOrUnset(
+      actions: EditorAction[],
+      unset: boolean,
+      unsetPaths: PropertyPath[],
+    ): EditorAction[] {
+      if (unset) {
+        return unsetPaths.map((path) => unsetProperty(element, path))
+      }
+      return actions
+    }
+
+    function shouldUnset(): boolean {
+      switch (e.type) {
+        case 'one-value':
+          return e.value == null
+
+        case 'two-value':
+          if (e.value.value != null) {
+            return false
+          }
+
+          let shorthandOtherDirections: CSSNumberOrNull[] = []
+          if (useShorthand) {
+            if (e.value.type === 'V') {
+              shorthandOtherDirections.push(aggregates.horizontal)
+            } else {
+              shorthandOtherDirections.push(aggregates.vertical)
+            }
+          }
+
+          return !useShorthand || shorthandOtherDirections.every((o) => o === null || o.value === 0)
```

Comment:
You can bin off the use of an array here completely
```suggestion
          if (useShorthand) {
            if (e.value.type === 'V') {
              return aggregates.horizontal === null || aggregates.horizontal.value === 0
            } else {
              return aggregates.vertical === null || aggregates.vertical.value === 0
            }
          } else {
            return true
          }
```

Reply:
oh nice :)